src/cli/templates.hh

```diff
@@ -1422,6 +1415,7 @@ version = 1.0.0
 
 
 [android]
+build_remove_path = dist/android/app/src/
```

Comment:
can we use the `settings["output"] + /android/app/src/` in cli.cc to not introduce extra config parameters?

Reply:
+1 if this is possible, I do this in some other places

Reply:
Seems to work fine, all done.